projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -2140,8 +2140,15 @@ function ( $option ) {
 		}
 
 		// File field requires Jetpack to be active
-		if ( $type === 'file' && ! defined( 'JETPACK__PLUGIN_DIR' ) ) {
-			return false;
+		if ( $type === 'file' ) {
+			/**
+			 * Check if Jetpack is active for file uploads.
+			 *
+			 * @since $$next-version$$
+			 *
+			 * @return bool
+			 */
+			return apply_filters( 'jetpack_forms_is_file_field_renderable', defined( 'JETPACK__PLUGIN_DIR' ) );
```

Comment:
do we still need this?

Reply:
We added it so that if someone uses the package only doesn't get the file upload block since it won't work without Jetpack.